init.lua

```diff
@@ -63,32 +72,70 @@ local function set_hud_overlay(player, name, show)
 end
 
 local function set_physics_overrides(player, overrides)
+	local player_name = player:get_player_name()
 	if has_player_monoids then
 		for name, value in pairs(overrides) do
 			player_monoids[name]:add_change(player, value, "hangglider:glider")
 		end
 	elseif has_pova then
-		pova.add_override(player:get_player_name(), "hangglider:glider",
+		pova.add_override(player_name, "hangglider:glider",
 				{jump = 0, speed = overrides.speed, gravity = overrides.gravity})
 		pova.do_override(player)
```

Comment:
TOCHECK: does pova really take a name for ``add_override`` and an object for ``do_override``?

Reply:
It actually does. https://codeberg.org/tenplus1/pova/src/commit/69adaf4e19550898aeea73ab0e1ee11f90a8ed6e/init.lua#L39 and https://codeberg.org/tenplus1/pova/src/commit/69adaf4e19550898aeea73ab0e1ee11f90a8ed6e/init.lua#L55